lib/analysis.js

```diff
@@ -90,7 +93,7 @@ function getQueryInfo(file, offset, type, start) {
         offset: offset
     }, function(error, data) {
         if (error) {
-            console.error("Error returned from Tern 'definition' request: " + error);
+            logger.log('error', "Error returned from Tern 'definition' request: " + error);
```

Comment:
I'd suggest to include stack trace if there was an error which *may* have stacktrace